src/app/containers/app-header/app-header.component.ts

```diff
@@ -161,6 +162,10 @@ export class AppHeaderComponent implements OnInit, OnDestroy {
       const applicationTheme: Theme = this.getApplicationThemeForUser(applicationThemes, userDetails.userInfo.roles);
       this.hideNavigationListener(this.store);
       this.setAppHeaderProperties(applicationTheme);
+      if (applicationTheme.navigationItems.some(x => x.active)) {
+        const currentNav = applicationTheme.navigationItems.find(x => x.active);
+        // this.router.navigate([currentNav.href]);
```

Comment:
Is this commenting out on purpose? Please remove if it is not going to be used again.

Reply:
Very good point. I have removed the redundant logic now. Thanks